en/notifications/README.md

```diff
@@ -10,23 +10,23 @@ For more information see [One Signal documentation](https//documentation.onesign
 
 ##### 2. Setup app Website Push Platforms in you app
 
-![](/media/notification-setup1.PNG)
+![](/notifications/media/notification-setup1.PNG)
```

Comment:
```suggestion
![notification setup](media/notification-setup1.PNG)
```

Reply:
Repeat for other image links.

See new [CONTRIBUTING](https://github.com/mautic/documentation/CONTRIBUTING.md) for updates on image links.